packages/components/src/mobile/bottom-sheet/color-cell.native.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+import { ColorIndicator } from '@wordpress/components';
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+import styles from './styles.scss';
+
+export default function BottomSheetColorCell( props ) {
+	const { onPress, color, ...cellProps } = props;
```

Comment:
I think the `onPress` doesn't need to be restructured and can be passed as one of `cellProps`